### ISSUE TITLE
vmauth cannot start with empty configuration.

### ISSUE THREAD

**Issue by f41gh7** [VictoriaMetrics/VictoriaMetrics#6467]
### Describe the bug

 vmauth reports error if configuration file is empty. It makes impossible to start vmauth if it's needed to deploy it now and any configuration later.
 Real world examples:
- helm charts
- victoria-metrics-operator

 Current workaround for it:
- provide "fake" config section record with random local port. Which potentially a security risk and may confuse users.
- launch nginx server with vmauth and return static stab error.

 It'd be great to accept empty configuration file and return `UnAuthorized` error for any request.

### To Reproduce

 Try to start vmauth with exists, but empty configuration file.

### Version

Any version of vmauth.

**Comment by f41gh7** [comment]
The feature was included into [v1.108.0](https://docs.victoriametrics.com/changelog/#v11080) release

**Comment by f41gh7** [comment]
Motivation for this change:
```
It may require to launch vmauth without configuration file.
Which is possible use case for operator based installations. Or any other installation, where launch and config generation of
vmauth is async operations. Mostly, it's needed for kubernetes based installations. 

One of such cases, is victoria-metrics-operator for Kubernetes. Which provides global resource VMAuth and allows to create
VMUser objects for it. Eventually operator creates configuration for VMAuth based on user defined selectors for VMUser.

Since there is no direct relations between
those objects. And any object could be created in on-demand by Kubernetes users. It's required to be able to start vmauth with empty auth config file.```
